src/components/adminComponents/layoutComponents/sidebar.jsx

```diff
@@ -60,7 +60,7 @@ import { IoMdClose } from "react-icons/io";
                   activeItem === item.name ? "bg-primary text-violet" : "text-white"
                 }`}
                 onClick={() => setActiveItem(item.name)}
-              ><Link  to={`/admin/${item.to}`}className="flex items-center w-full">
+              ><Link  to={item.to}className="flex items-center w-full">
```

Comment:
i updated in side bar files in develop brach please update the code, before giving this.. no need of this change